rust/core/src/driver_manager.rs

```diff
@@ -161,8 +170,20 @@ struct ManagedDriverInner {
     _library: Option<libloading::Library>,
 }
 
+#[derive(Default)]
+#[allow(dead_code)]
```

Comment:
```suggestion
```

Reply:
If we aren't using those fields maybe we should 1) add `pub` methods that return them 2) mark them `pub` or 3) remove them?

Reply:
updated to remove the fields but commented them out so we don't forget about them